x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -26,6 +26,9 @@ interface EventOptions {
   eventType?: string;
   eventCategory?: string | string[];
   processName?: string;
+  pid?: number;
+  parentPid?: number;
+  extensions?: object;
```

Comment:
This doesn't seem to be used. Please remove it if it isn't going to be used in this PR.

Reply:
per our convo: this is going to be used in testing. Please add a comment about intention.

Reply:
tests got found